### ISSUE TITLE
KinkMen

### ISSUE THREAD

**Issue by SailorSilly** [stashapp/CommunityScrapers#2049]
TLDR:
Issues with scraping scene using URL, date parses incorrect format.
Issues with scraping scene using fragments. Scene always comes back with "Hardcore Kinky & BDSM Porn Videos" as the results.

Are you scraping a scene, gallery, movie, or performer?
Scene

Scrape with URL? If so, what URLs have you tried?  

https://www.kinkmen.com/shoot/104072
https://www.kinkmen.com/shoot/104070
https://www.kinkmen.com/shoot/104020

The issue with the URL scraper is when trying to parse the date, it parses it incorrectly. It tries to pull in the format of: Jun 3, 2024.

Scrape with fragment? If so, what is the filename / title / studio code? 

The filename using the first URL above is: The Pleasure of Pain.mkv, title is: The Pleasure of Pain and studio code is: 104072

Whenever you try to scrape using a fragment, the results always return: Hardcore Kinky & BDSM Porn Videos. 

Please check the logs to see if there are any relevant error messages you can include

024-09-28 14:51:09Warning reading script result: json: unknown field "id"
2024-09-28 14:51:09Error   [Scrape / AnalVids] {"id": "64384", "title": "The Pleasure of Pain", "code": "104072", "url": "https://www.kinkmen.com/shoot/104072", "urls": ["https://www.kinkmen.com/shoot/104072"], "date": "2022-11-14", "details": "Sexy submissive stud Aiden Tyler starts out restrained to a St. Andrew\u2019s cross, blindfolded, ball-gagged and already leaking precum. Mason Wyler doesn\u2019t waste any time, and immediately begins cropping Aiden\u2019s body all over. After giving Aiden\u2019s nipples a hard pinch, Mason places a nipple clamp on each one. Loving the pain he is causing Aiden, Mason grabs him by the throat and makes out with him through the ball gag. Seeing that Aiden is still dripping precum, Mason decides the best solution is to plug his cock up with a urethral sound. Next, Aiden is strapped into a bondage chair completely naked; his legs spread exposing his hard cock. Mason enters and grabs a paddle and begins slapping Aiden all over, especially his cock. Despite all the pain he is receiving, Aiden is clearly loving it and Mason takes decides to take his torment in another direction. Using a vibrator, Mason edges Aiden while instructing him not to blow his load just yet. However, Aiden is unable to contain his pleasure, and makes a mess all over himself. As punishment, Mason delivers brutal post orgasm torment, slapping Aiden\u2019s cock and feeding him his own cum. Finally, Aiden is placed on his back with his wrists restrained to his ankles. Mason fucks Aiden\u2019s throat before spinning him around to use his ass until he cums deep inside Aiden!", "director": "Dominic Pacifico", "files": [{"id": "98164", "mod_time": "2024-09-03T11:08:20-04:00", "path": "/var/Farmers/Kink Men/104072 - The Pleasure of Pain (2022) Webrip-1080p/The Pleasure of Pain.mkv", "fingerprints": [{"type": "oshash", "fingerprint": "f932d08d17d5f4c0"}, {"type": "phash", "fingerprint": "b688479756cc8af2"}], "size": 1966985306, "format": "matroska", "width": 1920, "height": 1080, "duration": 3201.09, "video_codec": "hevc", "audio_codec": "aac", "frame_rate": 59.94, "bitrate": 4915791}]}
2024-09-28 14:51:09Error
